ZXing.Net.MAUI/ZXing.Net.MAUI.csproj

```diff
@@ -88,6 +88,6 @@
 	</Target>
 
 	<ItemGroup>
-	  <PackageReference Include="Microsoft.Maui.Controls" Version="8.0.82" />
+	  <PackageReference Include="Microsoft.Maui.Controls" Version="9.0.30" />
```

Comment:
I see some mixed MAUI versions between here and the other project. Should not be a big issue, but maybe lets align it? Or add it to a central place?